frontend/src/ConfirmationPage/ConfirmationPage.jsx

```diff
@@ -82,6 +89,7 @@ class ConfirmationPage extends React.Component {
       'Product Manager',
       'Other',
     ];
+    window.addEventListener('resize', this.handleResize);
```

Comment:
This isn't he right place to add a event listener please add a event listener inside `componentDidMount` and you need to run a cleanup function which removes the event listener inside `componentWillUnmount` lifecycle methods

Reply:
Thanks. Made the changes